google-cloud-speech/lib/google/cloud/speech/audio.rb

```diff
@@ -191,6 +191,8 @@ def url?
         #   phrases "hints" so that the speech recognition is more likely to
         #   recognize them. See [usage
         #   limits](https://cloud.google.com/speech/limits#content). Optional.
+        # @param [Boolean] words When `true`, return a list of words and
```

Comment:
As mentioned the naming (enabled_word_time_offsets) and documentation for this field should be made consistent with the proto.  e.g. note the use of "(timestamps)" in the description.

  // *Optional* If `true`, the top result includes a list of words and
  // the start and end time offsets (timestamps) for those words. If
  // `false`, no word-level time offset information is returned. 
  bool enable_word_time_offsets = 8;